docusaurus/docs/develop/envoy/introduction.md

```diff
@@ -363,13 +372,25 @@ make test_request_no_auth_header
 
 See the [Gateway Endpoint YAML File](#gateway-endpoint-yaml-file) section for more information on the `GatewayEndpoint` data structure.
 
+:::tip
```

Comment:
Remove. Duplicate of above. 

I think you rushed the merge too much.

I tried to take a "hollistic" view of the docs when reviewing and editing.

I do it every time putting myself in the shoes of someone looking at the docs for the first time.